Readme.md

```diff
@@ -0,0 +1,643 @@
+
+
+---
+
+#### get_commande_buffer_size
+**Return**:  
+- `int16_t`  
+
+**Parameters**: void  
+**Description**: number of command remaining (with the current command)  
+**Usage**: can be use to determine the free space in the command buffer if you want send a path with lot of got to point or angle. It can also be use to dertermine when the robot movement is finish  
+
+
+---
+
+#### get_direction_side
+**Return**:  
+- `Direction`  
+
+**Parameters**: void  
+**Description**: give the direction (NONE when the robot don't move backward or forward)  
+**Usage**: usefull to dermine wich side of the lidar point is usefull to look when you want avoid the enemy robot  
+
+
+---
+
+#### get_rotation_side
+**Return**:  
+- `Rotation`  
+
+**Parameters**: void  
+**Description**: give the rotation  
+
+
+---
+
+#### get_current_consigne
+**Return**: void  
+**Parameters**:  
+- `int16_t &x`  
+- ` int16_t &y`  
+- ` int16_t &theta`  
+- ` Type type`  
+
+**Description**: get current static consigne (angular consigne may change as robot moves forward or backward to compensate path variations)  
+
+
+---
+
+#### get_current_consigne
+**Return**:  
+- `Consigne`  
+
+**Parameters**: void  
+**Description**: Overloading  
+
+
+---
+
+#### get_moving_is_finish
+**Return**:  
+- `int16_t`  
+
+**Parameters**: void  
+**Description**: return true when the current consigne is finish (is equal to between get_running_is_finish or get_turning_is_finish)  
+**Usage**: when you pause the robot, to know when the robot is stop because you can't use get_commande_buffer_size. Otherwise, prefere to use get_commande_buffer_size.  
+
+
+---
+
+#### get_running_is_finish
+**Return**:  
+- `int16_t`  
+
+**Parameters**: void  
+**Description**: return true when the linear current consigne is finish  
+
+
+---
+
+#### get_turning_is_finish
+**Return**:  
+- `int16_t`  
+
+**Parameters**: void  
+**Description**: return true when the angular current consigne is finish  
+
+
+---
+
+#### get_linear_error
+**Return**:  
+- `int16_t`  
+
+**Parameters**:  
+- `int16_t &error`  
+
+**Description**: return the error between the dynamic consigne and the robot  
+
+
+---
+
+#### get_angular_error
+**Return**:  
+- `int16_t`  
+
+**Parameters**:  
+- `int16_t &error`  
+
+**Description**: return the error between the dynamic consigne and the robot  
+
+
+---
+
+#### get_current
+**Return**: void  
+**Parameters**:  
+- `int16_t &currentLeft`  
+- ` int16_t &currentLeft`  
+
+**Description**: gives the current average since the last reading for each motor  
+
+
+---
+
+#### get_speed
+**Return**: void  
+**Parameters**:  
+- `int16_t &speedLeft`  
+- ` int16_t &speedLeft`  
+
+**Description**: gives the speed average since the last reading for each motor  
+
+
+---
+
+## 5. MOTOR
+
+#### enable_motor
+**Return**: void  
+**Parameters**:  
+- `bool motorEnable`  
+
+**Description**: enable motor (the motor are disable at startup)  
```

Comment:
correction: enables the motors (disabled at startup)

Reply:
Meme en fait, puisquon a bool motorEnable, ce serait plutot set_motor_state(bool enabled)